### ISSUE TITLE
license not working

### ISSUE THREAD

**Issue by itsDantes** [bepass-org/oblivion#144]
it seems license option not working on v2, VPN getting discconected when we set a license key in app settings.

**Comment by Alireza-F-kenedy** [comment]
Lines was not working. I tested the new key. It worked. Test the new key

**Comment by itsDantes** [comment]
@Alireza-F-kenedy yeah. it seems its true, but why? is there any way to find if an warp+ key not working in true way?
